Gemfile.lock

```diff
@@ -309,7 +323,7 @@ GEM
     power_assert (2.0.3)
     psych (5.1.2)
       stringio
-    public_suffix (5.0.5)
+    public_suffix (6.0.0)
```

Comment:
Drops support for Ruby <3: https://github.com/weppos/publicsuffix-ruby/blob/main/CHANGELOG.md#600